docs/conventions/api/README.md

```diff
@@ -1 +1,24 @@
-To Do
+## RDS Query language(RQL) for Filtering, Sorting, and Pagination:
+
+We will be using a query string with a custom format for filtering and sorting, which closely following Github's search formats. You can read more about it [here](https://docs.github.com/en/search-github).
+
+#### Filtering:
+- Single search value: `?q=key:value`
+- Multiple search values: `?q=key:value1+key:value2`
+- Multiple key-value pairs: `?q=key1:value1+key1:value2+key2:value1+key2:value2`
+- Exclude a given value: `?q=-key:value`
+
+#### Sorting:
+- Ascending order: `?q=sort:key-asc`
+- Descending order: `?q=sort:key-desc`
+
+#### Pagination:
+- `size`: Number of items per page.
+- `page`: Navigate to a specific page.
+- `next`: Retrieve the next page.
```

Comment:
what values can come for `next` and `prev`, any examples can you share @Ajeyakrishna-k

Reply:
Any identifier which when passed, the server should give the next set of results. 
eg: next=abc

Reply:
got it